src/connectors/connector_cloudbuild.workflows.yaml

```diff
@@ -13,13 +13,15 @@
 # limitations under the License.
 
 # [START workflows_connector_cloudbuild]
-# This workflow demonstrates how to use the Cloud Build connector.
-# The workflow creates a simple build and the image is stored to an Artifact Registery repo. 
-# The workflow assumes a properly defined source tgz file already
-# exists in a Cloud Storage bucket: gs://your-project-id_cloudbuild/source/placeholder_src.tgz 
-# and a repository "your-project-id-docker-repo" already
-# exists in Artifact Registry where the image will be stored.
-# The new table and dataset are both deleted in the following steps.
+# This workflow demonstrates how to use the Cloud Build connector
+# to start a build that creates and pushes a conainer image to
+# an Artifact Registry repository.
+#
+# Prerequisites:
```

Comment:
This is great!

Reply:
I'm thinking use this as the basis for a docblock standard.